README.md

```diff
@@ -58,122 +52,20 @@ For Example:
-                ],                                                    
-                "Omega": [                                            
-                        "2.23E-02",                                   
-                        ".........",                                  
-                        "2.67E-02"                                    
-                ],                                                    
-                "Sigma": [                                            
-                        "........."                                   
-                ],                                                    
-                "OmegaCorr": [                                        
-                        "3.18E-02",                                   
-                        ".........",                                  
-                        "3.40E-02"                                    
-                ],                                                    
-                "SigmaCorr": [                                        
-                        "........."                                   
-                ]                                                     
-        },                                                            
-        "ParameterStructures": {                                      
-                "Theta": 5,                                           
-                "Omega": [                                            
-                        1,                                            
-                        0,                                            
-                        1                                             
-                ],                                                    
-                "Sigma": [                                            
-                        1                                             
-                ]                                                     
-        },                                                            
-        "ParameterNames": {                                           
-                "Theta": [                                            
-                        "KA",                                         
-                        "CL",                                         
-                        "V2",                                         
-                        "RUVp",                                       
-                        "RUVa"                                        
-                ],                                                    
-                "Omega": [],                                          
-                "Sigma": []                                           
-        },                                                            
-        "OFV": 2643.561                                               
-}    
+
```

Comment:
Both of these examples appear to be blank... Did you want to replace those with something or should we scrap this whole section?